### ISSUE TITLE
ESLint is still unable to resolve tsconfig `paths`

### ISSUE THREAD

**Issue by joaocasarin** [alexgorbatchev/eslint-import-resolver-typescript#103]
Error: `Unable to resolve path to module '@services/CreateUser'.eslint[import/no-unresolved](https://github.com/import-js/eslint-plugin-import/blob/v2.26.0/docs/rules/no-unresolved.md)`

Package dependencies:
```json
{
        "eslint": "^8.13.0",
        "eslint-config-airbnb-base": "^15.0.0",
        "eslint-config-prettier": "^8.5.0",
        "eslint-import-resolver-typescript": "^2.7.1",
        "eslint-plugin-import": "^2.26.0",
        "eslint-plugin-prettier": "^4.0.0",
        "typescript": "^4.6.3"
}
```

tsconfig.json paths:
```json
{
    "baseUrl": ".",
            "paths": {
                "@lambdas/*": ["src/lambdas/*"],
                "@services/*": ["src/services/*"]
            }
}
```

.eslintrc.js:
```js
module.exports = {
extends: ['airbnb-base', 'prettier'],
    parser: '@typescript-eslint/parser',
    settings: {
        'import/parsers': {
            '@typescript-eslint/parser': ['.ts', '.tsx']
        },
        'import/resolver': {
            node: {
                typescript: {
                    alwaysTryTypes: true,
                    project: './tsconfig.json'
                },
                extensions: ['.js', '.ts']
            }
        }
    },
    parserOptions: {
        ecmaVersion: 12,
        sourceType: 'module'
    },
    plugins: ['@typescript-eslint', 'import', 'prettier']
}
```

For the typescript build it is apparently good.

**Comment by JounQin** [comment]
`typescript` option should not be nested in `node`. (You don't need `node` for most cases.)

**Comment by joaocasarin** [comment]
> `typescript` option should not be nested in `node`. (You don't need `node` for most cases.)

Damn... I spent 1 hour trying to fix that and didn't notice that it was inside the `node` key... My apologies!

Regarding the need of `node`, I don't know why, but I tried removing it while trying to fix the eslint path thing and all my imports raised errors. Now that I put the typescript out of `node` and removed the extensions thing, it worked... Well, computers...

Thanks!
